src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -1139,802 +1131,802 @@ public abstract class X64
-        public static Vector128<double> Min(Vector128<double> left, Vector128<double> right) { throw null; }
-        public static Vector128<double> MinScalar(Vector128<double> left, Vector128<double> right) { throw null; }
-        public static int MoveMask(Vector128<byte> value) { throw null; }
-        public static int MoveMask(Vector128<sbyte> value) { throw null; }
-        public static int MoveMask(Vector128<double> value) { throw null; }
-        public static Vector128<long> MoveScalar(Vector128<long> value) { throw null; }
-        public static Vector128<ulong> MoveScalar(Vector128<ulong> value) { throw null; }
-        public static Vector128<double> MoveScalar(Vector128<double> upper, Vector128<double> value) { throw null; }
-        public static Vector128<ulong> Multiply(Vector128<uint> left, Vector128<uint> right) { throw null; }
-        public static Vector128<double> Multiply(Vector128<double> left, Vector128<double> right) { throw null; }
-        public static Vector128<double> MultiplyScalar(Vector128<double> left, Vector128<double> right) { throw null; }
-        public static Vector128<short> MultiplyHigh(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<ushort> MultiplyHigh(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
-        public static Vector128<int> MultiplyAddAdjacent(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<short> MultiplyLow(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<ushort> MultiplyLow(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
-        public static Vector128<byte> Or(Vector128<byte> left, Vector128<byte> right) { throw null; }
-        public static Vector128<sbyte> Or(Vector128<sbyte> left, Vector128<sbyte> right) { throw null; }
-        public static Vector128<short> Or(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<ushort> Or(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
-        public static Vector128<int> Or(Vector128<int> left, Vector128<int> right) { throw null; }
-        public static Vector128<uint> Or(Vector128<uint> left, Vector128<uint> right) { throw null; }
-        public static Vector128<long> Or(Vector128<long> left, Vector128<long> right) { throw null; }
-        public static Vector128<ulong> Or(Vector128<ulong> left, Vector128<ulong> right) { throw null; }
-        public static Vector128<double> Or(Vector128<double> left, Vector128<double> right) { throw null; }
-        public static Vector128<sbyte> PackSignedSaturate(Vector128<short> left, Vector128<short> right) { throw null; }
-        public static Vector128<short> PackSignedSaturate(Vector128<int> left, Vector128<int> right) { throw null; }
-        public static Vector128<byte> PackUnsignedSa
```

Comment:
This, and others, are missing the `new` modifier. Logged https://github.com/dotnet/arcade/issues/1488